testing/rkt/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Brian Rosner <brian@brosner.com>
+# Maintainer: Brian Rosner <brian@brosner.com>
+pkgname=rkt
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="container engine for Linux designed to be composable, secure, and built on standards"
+url="https://github.com/coreos/rkt"
+arch="x86_64"
+license="ASL 2.0"
+depends=""
+depends_dev=""
+makedepends="$depends_dev coreutils grep gzip diffutils cpio go bc linux-headers wget file gnupg autoconf automake bash openssl-dev acl-dev xz sed"
+subpackages=""
+source="rkt-$pkgver.tar.gz::https://github.com/coreos/rkt/archive/v$pkgver.tar.gz"
+builddir="$srcdir/rkt-$pkgver"
+
+build() {
+	cd "$builddir"
+	./autogen.sh
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+		--enable-tpm=auto \
+		--enable-sdjournal=auto \
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$builddir"
+	mkdir -p "$pkgdir"/var/lib/$pkgname
+	install -Dm755 "build-rkt-$pkgver"/target/bin/rkt "$pkgdir"/usr/bin/rkt
+	mkdir -p "$pkgdir"/usr/lib/rkt/stage1-images
```

Comment:
This line is unnecessary, `"$pkgdir"/usr/lib/rkt/stage1-images` is created on line 39 (`install -D`).